test_suite/conftest.py

```diff
@@ -33,6 +35,20 @@ def check_markers(host, request):
     host_version = host.system_info.release
     distro_version = f'{host_distro}{host_version}'
 
+    # Skip the test if the distro is explicitly excluded
+    if exclude_on_marker:
+        exclude_on_marker_list = exclude_on_marker.args[0]
+        if host_distro in exclude_on_marker_list \
+                or distro_version in exclude_on_marker_list:
+            pytest.skip(f'This test doesn't apply to {distro_version}')
```

Comment:
```suggestion
            pytest.skip(f"This test doesn't apply to {distro_version}")
```
Sorry my mistake, it's true that in this case the double quotes do make sense. I missed the apostrophe inside :sweat:

Reply:
Luckily you noticed! I haven't even try to challenge it since it made sense to me.. sneaky apostrophe!